understanding/20/three-flashes.html

```diff
@@ -12,11 +12,12 @@ <h1>Understanding Three Flashes</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Content does not trigger seizures</dd>
-         <dt>Author task</dt><dd>Do not flash content more than 3 times a second</dd>
+         <dt>Goal</dt><dd>Content does not trigger seizures.</dd>
+         <dt>What to do</dt><dd>Do not flash content more than 3 times a second.</dd>
+         <dt>Why it's important</dt><dd>Flashing content can cause seizures.</dd> 
```

Comment:
@rainbreaw commented
> In understanding/20/three-flashes-or-below-threshold.html and understanding/20/three-flashes.html consider expanding: "migraines, dizziness, nausea, or even seizures."
```suggestion
         <dt>Why it's important</dt><dd>Flashing content can cause migraines, dizziness, nausea, or even seizures.</dd> 
```

Reply:
Updating to match the adopted why language for the AA
```suggestion
         <dt>Why it's important</dt><dd>Flashing content can cause migraines, dizziness, nausea, and seizures.</dd> 
```